tf-psa-crypto/drivers/builtin/src/psa_crypto_ecp.h

```diff
@@ -175,6 +236,7 @@ psa_status_t mbedtls_psa_ecp_generate_key(
  * \retval #PSA_ERROR_CORRUPTION_DETECTED \emptydescription
  * \retval #PSA_ERROR_INSUFFICIENT_ENTROPY \emptydescription
  */
+
```

Comment:
```suggestion
```

In other entries there's no space between the documentation and the function's prototype, so I think we should follow the same pattern here.